src/common/components/orbit.h

```diff
@@ -4,37 +4,53 @@
 #pragma once
 
 #include <entt/entt.hpp>
-
+#include <math.h>
 #include "common/components/units.h"
+#include "spdlog/spdlog.h"
 
 namespace conquerspace {
 namespace common {
 namespace components {
-namespace bodies {
+namespace types {
 
 /**
  * Orbit of a body
  */
 struct Orbit {
-    types::degree theta;
-    types::astronomical_unit semiMajorAxis;
+    double gravitationalparameter;
```

Comment:
What is the gravitational parameter used for?

Reply:
It's critical for having slower orbit times around bodies with lower mass.
Eventually we might actually need to split this up further. (IE if we want to find surface gravity.